samples/subsys/usb/cdc_acm/boards/esp32s3_devkitm_procpu.overlay

```diff
@@ -0,0 +1,14 @@
+/*
```

Comment:
Obviously none of these board overlays are acceptable. But I am a bit surprised, did it not confuse you why there are no board overlays like in the samples?

Reply:
There are plenty of .overlay files for STM `nucleo` and Nordic boards hidden under the `samples/subsys/usb/...` Those overlays should be correctly placed under the `boards` folder too. 
However, what better location would you suggest for putting the endpoint nodes related to each USB example?